workspaces/rbac/plugins/rbac/package.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "@backstage-community/plugin-rbac",
+  "version": "1.32.0",
```

Comment:
latest version of janus plugin is 1.32.1 https://www.npmjs.com/package/@janus-idp/backstage-plugin-rbac